samples/Experimental/InputPlugin/main.cpp

```diff
@@ -0,0 +1,148 @@
+                                     if(inputSystem->GetMappingName(entity) == "Up" && (input.state == KeyState::KeyDown || input.state == KeyState::KeyDownHeld))
+                                     {
+                                         move += NovelRT::Maths::GeoVector2F{0.0f, 5.0f};
+                                     }
+                                     if(inputSystem->GetMappingName(entity) == "Left" && (input.state == KeyState::KeyDown || input.state == KeyState::KeyDownHeld))
+                                     {
+                                         move += NovelRT::Maths::GeoVector2F{-5.0f, 0.0f};
+                                     }
+                                     if(inputSystem->GetMappingName(entity) == "Right" && (input.state == KeyState::KeyDown || input.state == KeyState::KeyDownHeld))
+                                     {
+                                         move += NovelRT::Maths::GeoVector2F{5.0f, 0.0f};
+                                     }
+
+                                     //scale
+                                     if(inputSystem->GetMappingName(entity) == "A" && (input.state == KeyState::KeyDown || input.state == KeyState::KeyDownHeld))
+                                     {
+                                         scale += NovelRT::Maths::GeoVector2F{0.2f, 0.2f};
+                                     }
+                                     if(inputSystem->GetMappingName(entity) == "B" && (input.state == KeyState::KeyDown || input.state == KeyState::KeyDownHeld))
+                                     {
+                                         scale += NovelRT::Maths::GeoVector2F{-0.2f, -0.2f};
+                                     }
+
+                                     if(inputSystem->GetMappingName(entity) == "LeftClick" && (input.state == KeyState::KeyDown || input.state == KeyState::KeyDownHeld))
+                                     {
+                                         logger.logInfo("Clicked at {}, {}", input.mousePositionX, input.mousePositionY);
+                                     }
+                                 }
+
+                                 for (auto [entity, transform] : transforms)
+                                 {
+                                     TransformComponent newComponent{};
+                                     newComponent.scale = NovelRT::Maths::GeoVector2F::zero();
+                                     newComponent.scale += scale;
+                                     newComponent.positionAndLayer += move;
+                                     transforms.PushComponentUpdateInstruction(entity, newComponent);
+                                 }
+                             });
+
+    DummyUpdateStuff += [&](auto delta) { scheduler.ExecuteIteration(delta); };
+
+    logger.logInfoLine("\n\n\nPress the following:\nW - Move Up\nS - Move Down\nA - Move Left\nD - Move Right\nK - Grow\nL - Shrink");
```

Comment:
Is there no better way to handle the new lines here? This reads kinda oof.

Reply:
Split this into multiple logging lines.